src/lib/server/events/gcal.ts

```diff
@@ -1,36 +1,30 @@
-import * as RRule from 'rrule/dist/es5/rrule.min.js';
 import { Temporal } from '@js-temporal/polyfill';
 import { TEAMS } from '$lib/public/board/data';
-import { parseBool } from '$lib/server/parse-bool/parse-bool';
 import type { ClubEvent } from '$lib/public/events/event';
+import { google } from 'googleapis';
+import { config } from 'dotenv';
+import { destroy_each } from 'svelte/internal';
 
 export interface ICALParseOptions {
   filterBefore?: boolean;
   referenceDate?: Temporal.ZonedDateTime;
   maxEvents?: number;
 }
 
-export function parse(rawICAL: string, options?: ICALParseOptions): ClubEvent[] {
-  const events: ClubEvent[] = [];
-
+export async function fromGCal(
+  events: GCalEvent[],
+  options?: ICALParseOptions
```

Comment:
Omit options in `fromGCal`. Options are hardcoded in `listUpcomingEvents`.
<https://github.com/EthanThatOneKid/acmcsuf.com/blob/d282a7ef28a01774296cc5b5e42682014662d809/src/lib/server/events/gcal.ts#L319-L322>

Feel free to delete our old `ICALParseOptions` interface.